src/main/java/org/opensearch/knn/index/query/KNNQueryFactory.java

```diff
@@ -80,12 +80,7 @@ public static Query create(CreateQueryRequest createQueryRequest) {
         if (KNNEngine.getEnginesThatCreateCustomSegmentFiles().contains(createQueryRequest.getKnnEngine())) {
             if (filterQuery != null && KNNEngine.getEnginesThatSupportsFilters().contains(createQueryRequest.getKnnEngine())) {
                 log.debug(
-                    String.format(
-                        "Creating custom k-NN query with filters for index: %s \"\", field: %s \"\", " + "k: %d",
-                        indexName,
-                        fieldName,
-                        k
-                    )
+                    String.format("Creating custom k-NN query with filters for index: %s , field: %s , k: %d", indexName, fieldName, k)
```

Comment:
No need String.format here.
```suggestion
                    log.debug("Creating custom k-NN query with filters for index: {} , field: {} , k: {}", indexName, fieldName, k);
```

Reply:
done